js/assets/ohdsi.util.js

```diff
@@ -62,17 +62,13 @@ define(['jquery', 'knockout', 'lz-string', 'lodash', 'crossfilter'], function ($
 			return _isInitiallyDirty() || _initialState() !== ko.toJSON(root, _pruneJSON);
 		}).extend({
 			rateLimit: 200
-		});
+		});;
 
 		result.reset = function () {
 			_initialState(ko.toJSON(root, _pruneJSON));
 			_isInitiallyDirty(false);
 		};
 
-		result.makeDirty = function() {
```

Comment:
@chrisknoll, this was introduced during the work on the PR by @aklochkova, but I removed it during minor refactoring of her effort because I strongly thing that this is “hacky” way to do things and it should avoided until there exists a really good reason. In this case I do not see such reason and I was able to deal without this new method